modules/oe_theme_content_call_proposals/config/install/core.entity_view_display.node.oe_call_proposals.full.yml

```diff
@@ -0,0 +1,242 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.full
+    - field.field.node.oe_call_proposals.body
+    - field.field.node.oe_call_proposals.oe_call_proposals_contact
+    - field.field.node.oe_call_proposals.oe_call_proposals_deadline
+    - field.field.node.oe_call_proposals.oe_call_proposals_funding
+    - field.field.node.oe_call_proposals.oe_call_proposals_grants
+    - field.field.node.oe_call_proposals.oe_call_proposals_journal
+    - field.field.node.oe_call_proposals.oe_call_proposals_model
+    - field.field.node.oe_call_proposals.oe_call_proposals_opening_date
+    - field.field.node.oe_call_proposals.oe_departments
+    - field.field.node.oe_call_proposals.oe_documents
+    - field.field.node.oe_call_proposals.oe_publication_date
+    - field.field.node.oe_call_proposals.oe_reference_code
+    - field.field.node.oe_call_proposals.oe_summary
+    - node.type.oe_call_proposals
+  module:
+    - datetime
+    - entity_reference_revisions
+    - field_group
+    - link
+    - options
+    - rdf_skos
+    - text
+    - user
+third_party_settings:
+  field_group:
+    group_in_page_navigation:
+      children:
+        - group_item_details
+        - group_item_description
+        - group_item_documents
+        - group_item_contacts
+      parent_name: ''
+      weight: 0
+      format_type: oe_theme_helper_in_page_navigation
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Page contents'
+    group_item_details:
+      children:
+        - group_details
+        - group_results
+        - group_info
+      parent_name: group_in_page_navigation
+      weight: 20
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: Details
+    group_item_description:
+      children:
+        - body
+      parent_name: group_in_page_navigation
+      weight: 21
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: Description
+    group_item_documents:
+      children:
+        - oe_documents
+      parent_name: group_in_page_navigation
+      weight: 22
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: Documents
+    group_item_contacts:
+      children:
+        - oe_call_proposals_contact
+      parent_name: group_in_page_navigation
+      weight: 23
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: Contact
```

Comment:
This should be Contacts, it expects more items. You do it already with Documents.

Reply:
Field has label "Contacts" modules/oe_content_call_proposals/config/install/field.field.node.oe_call_proposals.oe_call_proposals_contact.yml but front-end label is "Contact" - it is determined in spec https://webgate.ec.europa.eu/fpfis/wikis/display/EWPP/Call+for+Proposals+Content+Type+Specification+-+Version+1.0.0

Reply:
The same case for Publication - https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=404786614.